packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -206,9 +181,11 @@ const Updates = decorate([
                 label: (
                   <span className='font-italic'>{_('unlistedChannel')}</span>
                 ),
-                value: UNLISTED_CHANNEL_VALUE,
+                value: '',
               },
             ],
+      channel: ({ _channel }, { xoaConfiguration }) =>
+        defined(_channel, xoaConfiguration.channel),
```

Comment:
Can you explain why this is better?

Reply:
In this PR we'll have only one defined value which will be changed by the UI and computed values which are only used for the display.

Reply:
Yes, but why is that better?

(Also, that's exactly what `helper` does, maybe you should have kept the existing style)

Reply:
I don't use a helper because it's the final value which it will be sent to `XOA_UPDATER` server.

Reply:
It's not necessary to send it if it's not been changed.